pages/about/index.vue

```diff
@@ -1,106 +1,73 @@
-                    $t('link.yycBlog2012')
-                }}</ext-link>
-            </template>
-            <template #linkYycBlog2013>
-                <ext-link :href="linkYycBlog2013" highlight>{{
-                    $t('link.yycBlog2013')
-                }}</ext-link>
-            </template>
-        </i18n>
-        <i18n
-            v-if="$t('paragraph.year2013')"
-            path="paragraph.year2013"
-            tag="p"
-            class="paragraph"
-        >
-            <template #linkYycBlog2013>
-                <ext-link :href="linkYycBlog2013" highlight>{{
-                    $t('link.yycBlog2013')
-                }}</ext-link>
-            </template>
-        </i18n>
-        <i18n path="paragraph.year2014And2015" tag="p">
-            <template #linkCheckIo>
-                <ext-link :href="linkCheckIo" highlight>CheckIO</ext-link>
-            </template>
-        </i18n>
-        <i18n path="paragraph.pyDay2015" tag="p"></i18n>
-        <i18n path="paragraph.year2016" tag="p">
-            <template #linkPyConBlog2016>
-                <ext-link :href="linkPyConBlog2016" highlight>{{
-                    $t('link.pyConBlog2016')
-                }}</ext-link>
-            </template>
-            <template #linkPhotoAlbum>
-                <ext-link :href="linkPhotoAlbum" highlight>{{
-                    $t('link.photoAlbum')
-                }}</ext-link>
-            </template>
-        </i18n>
-        <i18n path="paragraph.year2017" tag="p"></i18n>
-        <i18n path="paragraph.year2018" tag="p"></i18n>
-        <i18n path="paragraph.year2019" tag="p"></i18n>
-        <i18n path="paragraph.year2020" tag="p"></i18n>
-        <i18n path="paragraph.year2021" tag="p"></i18n>
-        <i18n path="paragraph.outro" tag="p"></i18n>
-    </i18n-page-wrapper>
+        </Banner>
+        <div class="flex flex-wrap justify-center mx-2 my-3">
+            <div v-for="(img, i) in introImgs" :key="`intro_img_${i}`">
+                <div
+                    class="w-32 md:w-56 lg:w-56 h-24 md:h-40 lg:h-40 bg-contain bg-no-repeat mx-2 my-3"
+                    :style="{
+                        'background-image': 'url(' + img + ')',
+                    }"
+                ></div>
+            </div>
+        </div>
+        <i18n-page-wrapper>
+            <p></p>
+        </i18n-page-wrapper>
+    </div>
 </template>
 
 <script>
 import i18n from '@/i18n/about/index.i18n'
 import I18nPageWrapper from '@/components/core/i18n/PageWrapper'
-import ExtLink from '@/components/core/links/ExtLink'
+import Banner from '@/components/core/layout/Banner'
+import CoreH1 from '@/components/core/titles/H1'
+import AboutBanner from '@/static/img/about/Banner.svg'
+import IntroImg1 from '@/static/img/about/Intro-1.png'
+import IntroImg2 from '@/static/img/about/Intro-2.png'
+import IntroImg3 from '@/static/img/about/Intro-3.png'
+import IntroImg4 from '@/static/img/about/Intro-4.png'
```

Comment:
Cool! I realized that this might solve the image routing discrepancy between local and deployment!